client/components/domains/register-domain-step/index.jsx

```diff
@@ -1163,7 +1161,6 @@ class RegisterDomainStep extends React.Component {
 						onButtonClick={ this.onAddDomain }
 						pendingCheckSuggestion={ this.state.pendingCheckSuggestion }
 						unavailableDomains={ this.state.unavailableDomains }
-						isEligibleVariantForDomainTest={ this.props.isEligibleVariantForDomainTest }
 						selectedFreePlanInSwapFlow={ this.props.selectedFreePlanInSwapFlow }
 						selectedPaidPlanInSwapFlow={ this.props.selectedPaidPlanInSwapFlow }
 						isReskinned={ this.props.isReskinned }
```

Comment:
For a site on the free plan, when I entered the launch flow, the initial suggestions view looks like this:

<img width="1285" alt="Screenshot 2020-09-23 at 12 38 42 PM" src="https://user-images.githubusercontent.com/1269602/93979711-4fdf1680-fd9b-11ea-9a6f-8cdfe6c8bc6b.png">

Notice the "Registration free" text for the domain price. The problem with showing this without the free domain explainer is that the domain price is misleading. The registration is free only if a paid plan is selected and this is explained in the free domain explainer box, but in the screenshot above it seems to indicate that registration is always free. We should either show the free domain explainer, or change the domain price text and I prefer the latter since showing the explainer in this view may not aesthetically fit in.

In the above line which is in `renderInitialSuggestions()`, can we add another prop to indicate this is an initial suggestion? Maybe something like `isInitialSuggestion={ true }`, and then [around here](https://github.com/Automattic/wp-calypso/pull/45618/files#diff-a9690a078a161112454a9d1639b75a58R48) you would show the registration free text only if `isSignupStep && ! isInitialSuggestion`. Maybe there is a better way to do it, just saying something top of my mind. 




can we add a prop isPlanSelectionUnavailableInFlow={ isPlanSelectionUnavailableInFlow } too? Then , render the "registration free" text only if plan selection

Reply:
Fixed at commit :  https://github.com/Automattic/wp-calypso/pull/45618/commits/ed504ab41af752dbbfedc8bc6e2e8b512a04759e#diff-99f1f1d8ba147cd06cfd4a0d8d195d34R592-R614